When the configure script is run with --disable-sound, it fails with:

checking for Wand... configure: error: The pkg-config script could not
be found or is too old. Make sure it is in your PATH or set the
PKG_CONFIG environment variable to the full path to pkg-config.

This is because the first use of the PKG_CHECK_MODULES macro in
configure.in is in the
if test "x$enable_sound" != "xno"
section, so is skipped if --disable-sound is given, causing the
pkg-config variables to not be initialized.

The pkg-config documentation warns about this and offers a fix:

Note that if there is a possibility the first call to
PKG_CHECK_MODULES might not happen, you should be sure to
include an explicit call to PKG_PROG_PKG_CONFIG in your
configure.ac

PKG_PROG_PKG_CONFIG([minversion])

Submitted upstream at:
https://sourceforge.net/tracker/?func=detail&aid=2767166&group_id=67131&atid=517001

diff -urp -x '*~' -x '*.orig' configure.in configure.in
--- configure.in	2009-04-13 07:12:07.000000000 -0700
+++ configure.in	2009-04-15 17:30:11.192993000 -0700
@@ -27,6 +27,7 @@ AC_PROG_INSTALL
 AM_DISABLE_SHARED
 AC_PROG_LIBTOOL
 AC_PROG_LN_S
+PKG_PROG_PKG_CONFIG
 
 # Checks for libraries.
 AC_CHECK_LIB([m], [pow],,AC_MSG_ERROR(-lm was not found but required.))
